zulipterminal/ui_tools/boxes.py

```diff
@@ -18,6 +18,8 @@ def __init__(self, view: Any) -> None:
         super(WriteBox, self).__init__(self.main_view(True))
         self.model = view.model
         self.view = view
+        self.is_edit_mode = False  # type: bool
+        self.msg_id = 0  # Defined when a message is edited
```

Comment:
It might be clearer in the text (and remove the need for this comment) if you named the variable differently?

At risk of combining two related states/meanings (though ensuring they're consistent), would it make sense to combine these two variables as an `Optional[int]` ?